src/plugins/discover/public/application/embeddable/saved_search_embeddable.tsx

```diff
@@ -86,6 +86,7 @@ export class SavedSearchEmbeddable
   private prevQuery?: Query;
   private prevSearchSessionId?: string;
   private searchProps?: SearchProps;
+  private isDestroyed?: boolean;
```

Comment:
I prefer to update the access modifier for the `destroyed` field from `private` -> `protected` instead of adding the duplicated `isDestroyed`

Reply:
done